dashboard/test/ui/features/contractEditorSections.feature

```diff
@@ -19,6 +19,7 @@ Scenario: Examples should be hidden when specified in contract editor
   And the "Definition" contract editor header is visible
 
   And I press "modalEditorClose"
+  And I press the last button with text "Ignore"
   And I configure the contract editor to disable examples
```

Comment:
Re: 1, I like that idea for these, I believe there is a toggle that can be set after initialization
Re: 2, I don't think it should animate like the instructions dialog.
Re: 3, could it be that your local apps or blockly-core code is out of date?